src/Core.ts

```diff
@@ -236,19 +236,13 @@ const replaceProjectName = (source: string): Program<string> =>
   pipe(
     RTE.ask<Environment>(),
     RTE.map(({ settings }) => {
-      const mkImportRegex = (projectName: string) => new RegExp(`from ('${projectName}')|("${projectName}")`, 'g')
-
-      // Matches imports of the form: `import { foo } from 'projectName'`
-      const root = mkImportRegex(settings.projectName)
-      // Matches imports of the form: `import { foo } from 'projectName/lib/...'`
-      const module = mkImportRegex(`${settings.projectName}/lib/`)
-      // Matches immports of the form: `import { foo } from 'projectName/...'`
-      const other = mkImportRegex(`${settings.projectName}/`)
-
-      return source
-        .replace(root, `from '../../${settings.srcDir}'`)
-        .replace(module, `from '../../${settings.srcDir}/`)
-        .replace(other, `from '../../${settings.srcDir}/`)
+      const importRegex = (projectName: string) =>
+        new RegExp(`from (?<quote>['"])${projectName}(?:\/lib)?(?:\/(?<path>.*))?\\k<quote>`, 'g')
```

Comment:
The regex uses named capturing groups, the `\\k<quote>` is a [backreference](https://javascript.info/regexp-backreferences#backreference-by-name-k).
The features are available since node 10, so I think this is safe.

Reply:
I agree - I think this is safe since we run our CI environment against Node 12.